packages/vite/src/template-tag.ts

```diff
@@ -15,9 +15,17 @@ export function templateTag({ inline_source_map } = { inline_source_map: false }
     name: 'embroider-template-tag',
     enforce: 'pre',
 
-    async resolveId(id: string, importer: string | undefined) {
+    async resolveId(id: string, importer: string | undefined, options) {
+      if (options.custom?.embroider?.isExtensionSearch) {
+        return null;
+      }
       let resolution = await this.resolve(id, importer, {
         skipSelf: true,
+        custom: {
```

Comment:
same with this one, you already have it added during the extension search so you just need to remove this one